build.gradle

```diff
@@ -137,11 +137,21 @@ allprojects {
 	}
 
 	jar {
+		manifest {
+			attributes 'Implementation-Title': 'FabricAPI',
+					'Implementation-Version': project.version,
+					'Main-Class': 'net.fabricmc.fabric.impl.launch.APIWarning'
+		}
 		classifier = "dev"
 	}
 
 	afterEvaluate {
 		remapJar {
+			manifest {
+				attributes 'Implementation-Title': 'FabricAPI',
+						'Implementation-Version': project.version,
+						'Main-Class': 'net.fabricmc.fabric.impl.launch.APIWarning'
+			}
```

Comment:
You should move this block into `tasks.withType(Jar)` so this implementation info and main class is available for any jar produced

Reply:
I think I've fixed both these, you mind reviewing again?